pom.xml

```diff
@@ -94,11 +94,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.0</version>
+        <version>3.3</version>
```

Comment:
I just copied whatever was on the error-prone instructions. I can update it, but i've already merged this into about 10 projects with these same versions... 😶